go.mod

```diff
@@ -9,19 +9,17 @@ replace (
 
 require (
 	github.com/BurntSushi/toml v0.3.1 // indirect
+	github.com/Microsoft/go-winio v0.4.12
 	github.com/aws/aws-sdk-go v1.23.20
 	github.com/go-git/go-git/v5 v5.2.0
-	github.com/gogo/protobuf v1.3.2 // indirect
 	github.com/google/uuid v1.1.1
 	github.com/kardianos/service v1.0.0
-	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
 	github.com/pkg/errors v0.8.1
 	github.com/prometheus/client_golang v1.1.0
 	github.com/prometheus/client_model v0.0.0-20190812154241-14fe0d1b01d4 // indirect
 	github.com/prometheus/procfs v0.0.4 // indirect
-	github.com/sirupsen/logrus v1.4.2
+	github.com/sirupsen/logrus v1.2.0
```

Comment:
is it downgraded?

Reply:
seems like it got downgraded by winio package. upgrading it back to 1.4.2